test/e2e/operator_test.go

```diff
@@ -1782,6 +1763,47 @@ func TestUniqueIdHeader(t *testing.T) {
 	}
 }
 
+// TestLoadBalancingAlgorithmUnsupportedConfigOverride verifies that the
+// operator configures router pod replicas to use the "leastconn" load-balancing
+// algorithm if the ingresscontroller is so configured using an unsupported
+// config override.
+func TestLoadBalancingAlgorithmUnsupportedConfigOverride(t *testing.T) {
+	icName := types.NamespacedName{Namespace: operatorNamespace, Name: "leastconn"}
+	domain := icName.Name + "." + dnsConfig.Spec.BaseDomain
+	ic := newPrivateController(icName, domain)
+	if err := kclient.Create(context.TODO(), ic); err != nil {
+		t.Fatalf("failed to create ingresscontroller: %v", err)
```

Comment:
Do we want to start using `%w` for errors in our e2e test code?

Reply:
I cannot use `%w` in `t.Fatalf`:

    test/e2e/operator_test.go:1775:3: Fatalf call has error-wrapping directive %w

However, I changed the `t.Errorf` call below to use `%w`.